### PR TITLE
Update look of the example app

### PR DESCRIPTION
**Pull request by danny-stytch**

### CODE REVIEW

File: public/home.html [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;700&family=IBM+Plex+Sans:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
     <link rel="stylesheet" href="styles.css">
   </head>
   <body>
     <div class="App">
-      <h2>You are logged in!</h4> 
+      <div class="header"></div>
```
Comment (mary-stytch) [inline public/home.html:13]: What is this empty div for?

File: public/styles.css [hunk 2]
```diff
@@ -1,14 +1,34 @@
+body {
+  background-color: #E5E5E5;
+}
+
+.header {
+  position: absolute;
+  height: 59px;
+  width: 100%;
+  left: -1px;
```
Comment (mary-stytch) [inline public/styles.css:9]: We should be able to achieve this without the top and left values here. What problem are these values solving?

File: public/home.html (hunk 1 above)
Comment (danny-stytch) [reply public/home.html:13]: It's meant to be the nav bar at the top. I see that the stytchjs-react app uses a <header> HTML tag to do this. Should I change it to use that?

File: public/styles.css [hunk 3]
```diff
@@ -1,14 +1,34 @@
+body {
+  background-color: #E5E5E5;
+}
+
+.header {
+  position: absolute;
```
Comment (mary-stytch) [inline public/styles.css:6]: style nit (and I know I didn't do this with the original css) please alphabetize the css values within each object. Thanks!

File: public/styles.css [hunk 4]
```diff
@@ -1,14 +1,34 @@
...
+  background-color: #FFFFFF;
+  width: 450px;
+  height: 375px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #EBEBEB;
+  box-sizing: border-box;
+  box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.12);
+  border-radius: 9px
```
Comment (mary-stytch) [inline public/styles.css:33]:
```suggestion
  border-radius: 9px;
```

File: public/styles.css (hunk 3 above)
Comment (danny-stytch) [reply public/styles.css:6]: To confirm,  should something like ` box-sizing: border-box;` be changed to `box-sizing: Border-Box;`?

File: public/styles.css (hunk 2 above)
Comment (danny-stytch) [reply public/styles.css:9]: The left value can be removed. Removing the top results in the following
image

File: public/styles.css (hunk 4 above)
Comment (danny-stytch) [reply public/styles.css]: Fixed here: https://github.com/stytchauth/stytchjs-html-express/pull/9/commits/9b1e5294bb4126ddec94257368ed67a2783165f7#diff-c0957aceb7c1368cd9919f48a72240ae31b22112283b1bd4d38c66a55ed584b9R32

File: public/home.html (hunk 1 above)
Comment (danny-stytch) [reply public/home.html]: I just went ahead and switched it to a <header> tag here: https://github.com/stytchauth/stytchjs-html-express/pull/9/commits/9b1e5294bb4126ddec94257368ed67a2783165f7#diff-380e384daa5a30a47fcabfd90df4dbd52d59e9b03fae35c1368ec7f8c7390edfR11

File: public/styles.css (hunk 3 above)
Comment (mary-stytch) [reply public/styles.css]: no, all lower case is the convention in CSS so `box-sizing: border-box;` is fine.

File: public/styles.css [hunk 5]
```diff
@@ -1,14 +1,33 @@
...
+  background-color: #E5E5E5;
+}
+
 .App {
-  text-align: center;
-  font-family: "Helvetica New", Helvetica, sans-serif;
-  display: flex;
-  justify-content: center;
-  align-items: center;
+  text-align: Center;
```
[the remark this answers is not in the document]
Comment (danny-stytch) [reply public/styles.css:6]: 🤦‍♂️, no, this is my bad. I'll change it back

[the remark this answers is not in the document]
Comment (danny-stytch) [reply public/styles.css]: K I reverted that change and alphabetized it here: https://github.com/stytchauth/stytchjs-html-express/pull/9/commits/2c2341727bfcc17703354d5b636f8b0a79262ee4

File: public/home.html [hunk 6]
```diff
@@ -2,11 +2,16 @@
 <html lang="en">
   <head>
     <title>Home | Stytch example</title>
+    <link
```
Comment (mary-stytch) [inline public/home.html:5]: Please remove this font link since we'll be using Helvetica now.

File: public/signupOrLogin.html [hunk 7]
```diff
@@ -3,12 +3,13 @@
   <head>
     <title>Login | Stytch example</title>
     <link rel="stylesheet" href="styles.css">
+
```
Comment (mary-stytch) [inline public/signupOrLogin.html:6]:
```suggestion
```

File: public/signupOrLogin.html [hunk 8]
```diff
@@ -42,13 +43,15 @@ <h2>Sign up or log in</h4>
         }
       });
       var style = {
-        fontFamily: '"Helvetica New", Helvetica, sans-serif',
+        fontFamily: 'Helvetica',
```
Comment (mary-stytch) [inline public/signupOrLogin.html:46]: Let's use this full font family name.
```suggestion
        fontFamily: '"Helvetica New", Helvetica, sans-serif',
```

File: public/styles.css [hunk 9]
```diff
@@ -1,14 +1,33 @@
...
+}
+
 .App {
-  text-align: center;
-  font-family: "Helvetica New", Helvetica, sans-serif;
-  display: flex;
-  justify-content: center;
   align-items: center;
+  display: flex;
+  font-family: 'Helvetica';
```
Comment (mary-stytch) [inline public/styles.css:8]:
```suggestion
  font-family: "Helvetica New", Helvetica, sans-serif;
```

**Review by mary-stytch — APPROVED** [review]
Just a few suggestions, looks good otherwise!

**Pull request merged**